src/k8s-engine/client.ts

```diff
@@ -215,6 +215,10 @@ export class KubeClient extends events.EventEmitter {
     this.services = null;
   }
 
+  get k8sClient() {
```

Comment:
This doesn't seem to actually return the client; maybe call this `get config`?

Reply:
It's a client as used in https://github.com/rancher-sandbox/rancher-desktop/blob/6170143754e83ccc24d24c497208cfc1e8f95b04/src/k8s-engine/images/imageProcessor.ts#L363